src/js/_enqueues/admin/common.js

```diff
@@ -2220,8 +2220,14 @@ $( function( $ ) {
 	// Listen for jQuery AJAX events.
 	( function( $ ) {
 		$( document ).ajaxComplete( function( event, xhr, settings ) {
+
+			// Return early if this is not the plugin-install.php page.
+			if ( window.pagenow !== 'plugin-install' ) {
+				return;
+			}
+
 			// Check if this is the 'search-install-plugins' request.
-			if ( settings.data && settings.data.includes( 'action=search-install-plugins' ) ) {
+			if ( settings.data && typeof settings.data === "string" && settings.data.includes( 'action=search-install-plugins' ) ) {
```

Comment:
```suggestion
			if ( settings.data && typeof settings.data === 'string' && settings.data.includes( 'action=search-install-plugins' ) ) {
```
Just to keep consistency, I think you can use 'string' instead of "string" because the single quote is a more strict sting and everywhere uses a single quote of this file.

Reply:
@rudlinkon Thank you for reviewing the PR. 
I've updated it as per your suggestion